src/_scss/components/header/_headerSubMenu.scss

```diff
@@ -0,0 +1,43 @@
+@mixin usa-nav-submenu {
+  @include usa-sidenav-sublist;
+
+  @include media($nav-width) {
+    @include unstyled-list;
+    @include padding(1.15rem null);
+    background-color: $color-primary-darkest;
+    min-width: 21.5rem;
+    width: auto;
+    position: absolute;
+  }
+
+  &[aria-hidden=true] {
```

Comment:
I see Aria being used elsewhere in a similar way; if this our code, we should rethink its use.